Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -143,6 +165,12 @@ public static void VerifySteamID(string steamSessionTicket, Action<LootLockerVer
             LootLockerAPIManager.Verify(verifyRequest, onComplete);
         }
 
+        /// <summary>
+        /// Convert a steam ticket so LootLocker can read it.
+        /// </summary>
+        /// <param name="ticket"></param>
+        /// <param name="ticketSize"></param>
```

Comment:
Is it obvious for our users what the size should be set to? Because it isn't to me with 0 insight ;P Worth adding a not on what it means?

Reply:
The steam ticket and ticket-size is decided by Steam, and not by us. We have documentation for it, so will link to that.